client/src/components/SourcesList/SourcesList.tsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { request, gql } from 'graphql-request';
+import { Link } from 'react-router-dom';
+import { Urls } from '../../lib/urls';
+
+
+export function SourcesList() {
+  const [sources, setSources] = useState([]);
+  useEffect(() => {
+    (async () => {
```

Comment:
its way more readable if you define a funciton like

```
const foo = async () => {
}
foo()
```
rather than immediately invoking